### DISCUSSION TITLE
Test discussion

### DISCUSSION THREAD

**Discussion by Aranud** [etlegacy/etlegacy#1625] [General]
A nice text with a few words.

**Comment by isRyven** [comment]
:baguette_bread: :wine_glass: :frog: :tokyo_tower: :heart:

**Reply by isRyven** [reply]
rip discord

**Reply by Aciz** [reply]
> rip discord

come back senpai

**Reply by Aranud** [reply]
> 
> 
> rip discord

Sir, where are you ?

**Reply by ghost** [reply]
Very nice to have this feature here! Btw, there are alternative messengers to Discord ...

**Comment by Aranud** [comment]
@etlegacy/developers does this thing work too ?

**Reply by isRyven** [reply]
Yes it works.
